src/Glpi/Config/LegacyConfigurators/ConfigRest.php

```diff
@@ -94,10 +94,12 @@ public function execute(): void
         // The user's current groups are stored in his session
         // If there was any change regarding groups membership and/or configuration, we
         // need to reset the data stored in his session
-        $last_group_change = $GLPI_CACHE->get('last_group_change');
         if (
             isset($_SESSION['glpigroups'])
-            && ($_SESSION['glpigroups_cache_date'] ?? "") < $last_group_change
+            && (
+                !array_key_exists('glpigroups_cache_date', $_SESSION)
```

Comment:
A call to `isset($_SESSION['glpigroups_cache_date'])` would be even faster, is there a reason for `array_key_exists` ?

Reply:
I used to use `array_key_exists` because I already had to fix some bugs in the past related to the fact that considering `null` value as missing was an issue.